backend/src/routes/users.js

```diff
@@ -2,56 +2,59 @@ const router = require('express').Router()
 const knex = require('../database/connection')
 const passport = require('passport')
 const utils = require('../lib/utils')
+const path = require('path')
+const filename = path.basename(__filename)
+const Logger = require('../lib/logger')(filename)
 
 router.get('/protected', passport.authenticate('jwt', { session: false }), (req, res, next) => {
     res.status(200).json({ success: true, msg: 'You are successfully authenticated to this route!' })
 })
 
 // Validate an existing user and issue a JWT
-router.post('/login', function (req, res, next) {
+router.post('/login', async function (req, res, next) {
+    Logger.info(`/login`)
     const { email, password } = req.body
-    console.log('req body: ', email)
 
-    knex('users')
-        .where('email', email)
-        .limit(1)
-        .then((response) => {
-            const user = response[0]
-            if (!user) {
-                return res.status(401).json({ success: false, msg: 'could not find user' })
-            }
+    try {
+        const user = await knex('users').where('email', email).first()
 
-            const isValid = utils.validPassword(password, user.hash, user.salt)
+        if (!user) {
+            Logger.info(`User unavailable for email - ${email}`)
```

Comment:
this is nice

Reply:
in logger.js, there's this line..

`const level = () => {
    const env = process.env.NODE_ENV || 'development'
    const isDevelopment = env === 'development'
    return isDevelopment ? 'debug' : 'warn'
}`

so currently in production, it'll still display errors/warnings and I believe even write to a file only for errors due to this line
`new winston.transports.File({
        filename: 'logs/error.log',
        level: 'error',
        format: winston.format.json(),
    }),
    `